rollup.config.js

```diff
@@ -63,11 +63,14 @@ if (devMode || demoMode) {
 }
 
 if (devMode) {
-  // Create a server for dev mode
-  plugins.push(serve({ contentBase: 'dist', port: 10001 }));
-  // Allow to livereload on any update
   if (devLiveReloadMode) {
+    // Create a server for dev live reload mode
```

Comment:
:question: could we manage this by passing an argument when calling npm run start/watch to specifiy the port?
This would avoid hardcoding the port here and have a special branch

Reply:
Yes, I didn't think to do that.